src/app/business/common/components/detail/index.js

```diff
@@ -37,7 +37,7 @@ class Detail extends Component {
         return (
             <PanelWrapper>
                 <PanelTop className={css`justify-content: space-between;`}>
-                    <Title item={item} />
+                    <Title item={item} className={css`text-overflow: ellipsis; overflow: hidden;`} />
```

Comment:
Could you move this to a const?

Reply:
I moved it directly in the "Title" component, this it is only used here.